app/web/features/search/FilterDialog.tsx

```diff
@@ -255,9 +253,7 @@ export default function FilterDialog({
                 id="num-guests-filter"
                 value={numberOfGuestFilter}
                 onChange={(e) => {
-                  setNumberOfGuestFilter(
-                    e.target.value as unknown as undefined
-                  );
+                  setNumberOfGuestFilter(e.target.value);
```

Comment:
😮 no ts complains?

Reply:
It's because I set the type in the useState variable it references so now it matches :-)